LiteFarmOrg/LiteFarm
# Improved browser compatability of our map download

This new approach should have much better browser compatibility than before.

To test:
1. Go to your farm map
2. Click the download button
3. Click the "Donwload now" button
4. The map should download successfully in your browser
